server/src/main/java/io/crate/planner/CreateUserMappingPlan.java

```diff
@@ -38,10 +42,25 @@
 
 public class CreateUserMappingPlan implements Plan {
 
+    private static final Set<String> SUPPORTED_OPTIONS = Set.of("user", "password");
```

Comment:
The available set of options depends on the concrete ForeignDataWrapper implementation. We can't hardcode this to  user/password for all of them.

This should be done in the same way as it was done for create server and create foreign table.
See https://github.com/crate/crate/blob/ddef9b04a188251a375aacef9f6a2411ff8a78ad/server/src/main/java/io/crate/planner/CreateServerPlan.java#L76-L77

Reply:
Thank you, initially assumed that all implementations required those. Working on a separate PR https://github.com/crate/crate/pull/15606 following CreateServerPlan.